pro/controllers/users.go

```diff
@@ -1279,58 +1280,102 @@ func getUserRemoteAccessGwsV1(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	userGwNodes := proLogic.GetUserRAGNodes(*user)
+
```

Comment:
Should check this logic
I believe this change was causing issues, so we had to revert the change from the release patch.